app/controllers/users_controller.rb

```diff
@@ -16,16 +16,17 @@ def new
   end
 
   def edit
+    authorize @user
     if @user == current_user
       redirect_to edit_user_registration_path
     else
       set_form_dropdowns
     end
   end
 
+  # NOTE: I'm not entirely sure this is used at all.
```

Comment:
yeah, i'm considering a new controller so admins can do create user+person in one step and removing this from here bc it's not used/useable

Reply:
I can't get the tests for it to run,  or even route to it so that I can insert a byebug! So I'm going to remove it.